src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -518,4 +520,32 @@ def remove_file_post
       end
     end
   end
+
+  describe 'GET #trigger_services' do
+    let(:post_url) { "http://localhost:3200/source/#{source_project}/#{service_package}?cmd=runservice&user=#{user}" }
```

Comment:
And if it doesn't work I would prefer to only have one ``let`` statement with just the base url and concatenate it with path in the test.